pkg/lib/accountmanagement/service.go

```diff
@@ -40,9 +53,89 @@ type FinishAddingInput struct {
 type FinishAddingOutput struct {
 	// It is intentionally empty.
 }
+type ResendOTPCodeInput struct {
+	Channel      model.AuthenticatorOOBChannel
+	LoginID      string
+	isSwitchPage bool
+}
+
+type sendOTPCodeInput struct {
+	Channel  model.AuthenticatorOOBChannel
+	Target   string
+	isResend bool
+}
+
+type StartCreateIdentityWithVerificationInput struct {
+	LoginID    string
+	LoginIDKey string
+	Channel    model.AuthenticatorOOBChannel
+}
+
+type StartUpdateIdentityWithVerificationInput struct {
+	LoginID    string
+	LoginIDKey string
+	IdentityID string
+	Channel    model.AuthenticatorOOBChannel
+}
+
+type startIdentityWithVerificationInput struct {
+	LoginID    string
+	LoginIDKey string
+	IdentityID string
+	Channel    model.AuthenticatorOOBChannel
+	isUpdate   bool
+}
+
+type StartIdentityWithVerificationOutput struct {
+	Token            string `json:"token,omitempty"`
```

Comment:
json tag should be all or nothing. I think at this stage we don't need to add those